src/api/index.ts

```diff
@@ -14,26 +23,31 @@ const isIFrame = (() => {
     }
 })();
 
-export function enable(themeOptions: Partial<Theme> = {}, fixes: DynamicThemeFix = null) {
+export function enable(themeOptions: Partial<Theme> = {}, fixes: DynamicThemeFix = {} as DynamicThemeFix) {
     const theme = {...DEFAULT_THEME, ...themeOptions};
 
     if (theme.engine !== ThemeEngines.dynamicTheme) {
         throw new Error('Theme engine is not supported');
     }
 
+
     createOrUpdateDynamicTheme(theme, fixes, isIFrame);
+    const frames = document.getElementsByTagName("iframe");
+    const script = document.createElement('script');
+    script.src = 'https://cdn.jsdelivr.net/npm/darkreader@latest/darkreader.min.js'
+    const scriptEnable = document.createElement('script');
+    scriptEnable.textContent = `DarkReader.enable()`;
```

Comment:
FYI, the script will be executed before the remote script. There are several options like `defer` etc.